### PR TITLE
chore: update CSV description

### PR DESCRIPTION
**Pull request by codingben**
**What this PR does / why we need it**:

Update CSV description to be similar as we have
in README.md.

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by codingben** [comment]
Should we backport this PR to previous releases?

File: config/manifests/bases/ssp-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
     capabilities: Basic Install
     categories: Openshift Optional
     containerImage: REPLACE_IMAGE:TAG
-    description: Manages KubeVirt addons for Scheduling, Scale, Performance
+    description: Deploying and controlling additional KubeVirt resources
```
Comment (jcanocan) [inline config/manifests/bases/ssp-operator.clusterserviceversion.yaml:9]: What do you think about "Deploys and controls additional resources"?

File: config/manifests/bases/ssp-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
       kind: SSP
       name: ssps.ssp.kubevirt.io
       version: v1beta2
-  description: KubeVirt Schedule, Scale and Performance Operator
+  description: Operator that deploying and controlling additional KubeVirt resources
```
Comment (jcanocan) [inline config/manifests/bases/ssp-operator.clusterserviceversion.yaml:21]: What about "Kubevirt Operator for deploying and controlling additional Kubevirt resources"

File: data/olm-catalog/ssp-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -38,7 +38,7 @@ metadata:
     capabilities: Basic Install
     categories: Openshift Optional
     containerImage: REPLACE_IMAGE:TAG
-    description: Manages KubeVirt addons for Scheduling, Scale, Performance
+    description: Deploying and controlling additional KubeVirt resources
```
Comment (jcanocan) [inline data/olm-catalog/ssp-operator.clusterserviceversion.yaml:41]: Same as above.

File: data/olm-catalog/ssp-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -57,7 +57,7 @@ spec:
       kind: SSP
       name: ssps.ssp.kubevirt.io
       version: v1beta2
-  description: KubeVirt Schedule, Scale and Performance Operator
+  description: Operator that deploying and controlling additional KubeVirt resources
```
Comment (jcanocan) [inline data/olm-catalog/ssp-operator.clusterserviceversion.yaml:60]: Same as above.

**Review by jcanocan — COMMENTED** [review]
Thanks for pushing this effort forward. I've added some comments, fell free to ignore them if you don't agree.

File: config/manifests/bases/ssp-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (0xFelix) [reply config/manifests/bases/ssp-operator.clusterserviceversion.yaml:21]: `Operator that manages additional KubeVirt related resources`? Or at least `Operator that deploys and controls additional KubeVirt resources`

**Comment by jcanocan** [comment]
Looks good to me! Thanks, @codingben!

**Review by 0xFelix — APPROVED** [review]
/approve
/lgtm

**Pull request merged**